sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -342,10 +338,14 @@ export class ReceiveHandler {
 export { RetryOptions }
 
 // @public
-export interface SendOptions {
+export interface SendOptions extends SendOptionsBase {
```

Comment:
What is the reason to add a base for send options?

Reply:
This was to make the setting of partitionKey a compile error since it conflicts with sending batches (partitionKey for batches can only be set at the batch level).

Reply:
In that case I would suggest the below
- rename `SendOptionsBase` to `SendBatchOptions`
- don't export `SendOptions` as it is used for individual send which is no longer exposed to the user

Reply:
Done.